Makefile

```diff
@@ -36,7 +56,9 @@ all build:
 #   make check
 #   make test
 #   make check WHAT=pkg/build GOFLAGS=-v
-check test:
+check-test: export STI_COVER= -cover -covermode=atomic
+check-test: export STI_RACE=  -race
+check-test: verify
```

Comment:
I've copied that solution from origin/k8s, I'm not a makefile expert but a bit of googling gave me [Target-specific Variable Values](https://www.gnu.org/software/make/manual/html_node/Target_002dspecific.html), that's why we have separate for each value. But you're right the target name shouldn't be changed.